src/server/db_setup.ts

```diff
@@ -182,3 +181,63 @@ export async function insertGameResults(
     // TODO: Could return latest DB results here
     return { status: 200, results: [] };
 }
+
+// Checks the existence of an ID in the database
+export async function checkGameIdExists(
+    gameId: string
+): Promise<boolean>{
+  // Check if the game already exists in the table
+  const res = await pool.query(
+      "SELECT * FROM game_results WHERE game_label = $1",
+      [gameId]
```

Comment:
We could make this more efficient by taking in a list of gameIds, and making the query `"... WHERE game_label IN $1", [gameIds]`
Though I'm not sure how our postgres library handles list types... so if that doesn't immediately work, don't worry about it.

Reply:
Added TODO to address down the line, will look into soon.